### PR TITLE
feat: org unit condition (TECH-883)

### PR DESCRIPTION
**Pull request by martinkrulltott**
Implements [TECH-883](https://jira.dhis2.org/browse/TECH-883)

---

### Key features

1. Implements the org unit condition

---

### Description

Adds support for `valueType: ORGANISATION_UNIT`. Also fixes the scrollbar for the conditions modal so that only the condition content scrolls (everything within the gray box, i.e. not the text description at the top).

---

### Screenshots

image

### CODE REVIEW

**Comment by martinkrulltott** [comment]
FYI, to debug this in Chrome dev tools, the ’Show user agent shadow DOM’ setting needs to be enabled https://stackoverflow.com/a/26853319/1581477

File: src/components/Dialogs/Conditions/ConditionsManager.js [hunk 1]
```diff
@@ -52,6 +53,7 @@ const DIMENSION_TYPE_TRUE_ONLY = 'TRUE_ONLY'
 const DIMENSION_TYPE_DATE = 'DATE'
 const DIMENSION_TYPE_TIME = 'TIME'
 const DIMENSION_TYPE_DATETIME = 'DATETIME'
+const DIMENSION_TYPE_ORGANISATION_UNIT = 'ORGANISATION_UNIT'
```
Comment (janhenrikoverland) [inline src/components/Dialogs/Conditions/ConditionsManager.js:56]: In dhis2 terminology these are "value type", but probably makes sense from an ER pov.

File: src/components/Dialogs/Conditions/OrgUnitCondition.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    addMetadata,
+    addParentGraphMap,
+    condition,
+    onChange,
+    rootOrgUnits,
+    metadata,
+    parentGraphMap,
+}) => {
+    const parts = condition.split(':')
+    const value = parts[1]?.length && parts[1]
```
Comment (janhenrikoverland) [inline src/components/Dialogs/Conditions/OrgUnitCondition.js:26]: Pretty simple so maybe not necessary, but ideally a function with a unit test.

**Review by janhenrikoverland — APPROVED** [review]
LG

**Pull request merged**